hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -649,12 +649,25 @@
 
     // CLKMGR tests:
     {
-      name: chip_sw_clk_off_trans
+      name: chip_sw_clk_idle_trans
       desc: '''Verify the ability to turn off the transactional clock via SW.
 
-             Ensure that activity in any of the IPs running on this clock prevents the clock from
-             actually being turned off. Verify that turning off this clock does not affect the other
-             derived clocks.
+            Ensure that activity in any of the IPs running on this clock prevents the clock from
+            actually being turned off until the activity is completed, at which point the clock is
+            turned off.  Verify it is off via spinwait in hints_status CSR, and then check the state
+            of the transactional unit to correspond to a completed activity.  Verify that turning
```

Comment:
this should also cover blocks (such as otbn) with multiple idles right?

Reply:
Yes it should fully cover all bits of `clkmgr:idle_i`.

Reply:
Agreed: and all bits of `clkmgr_hints`.